masu/external/accounts/network/cur_accounts_network.py

```diff
@@ -0,0 +1,114 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Database source impelmentation to provide all CUR accounts for CURAccounts access."""
+
+import requests
+
+from masu.config import Config
+from masu.external.accounts.cur_accounts_interface import (CURAccountsInterface,
+                                                           CURAccountsInterfaceError)
+
+
+# pylint: disable=too-few-public-methods
+class CURAccountsNetwork(CURAccountsInterface):
+    """Provider interface defnition."""
+
+    def __init__(self):
+        """Initializer."""
+        self.koku_host = Config.KOKU_HOST
+        self.koku_port = str(Config.KOKU_PORT)
+        self.admin_user = Config.KOKU_ADMIN_USER
+        self.admin_pass = Config.KOKU_ADMIN_PASS
+
+    def _get_service_admin_token(self):
+        """
+        Retrieve service admin token from Koku.
+
+        This will call koku's token-auth to get the service admin token.
+
+        Args:
+            None
+
+        Returns:
+            (String) : Service Admin token
+
+        """
+        token_auth_url = 'http://{}:{}/api/v1/token-auth/'.format(self.koku_host, self.koku_port)
+        data = {'username': self.admin_user, 'password': self.admin_pass}
+
+        try:
+            response = requests.post(token_auth_url, data=data)
+            if response.status_code != 200:
```

Comment:
Since this is masu, and we're using Flask, I don't think we should be adding a django rest framework dependency to this repo in order to use its constants. requests returns `200` and I think comparing that directly to `200` is okay.